src/Apps/Sell/Routes/__tests__/ThankYouRoute.jest.tsx

```diff
@@ -0,0 +1,15 @@
+import { ThankYouRoute } from "Apps/Sell/Routes/ThankYouRoute"
+import { mount } from "enzyme"
+
+describe("ThankYouRoute", () => {
+  it("renders text and links", () => {
+    const wrapper = mount(<ThankYouRoute />)
+    const text = wrapper.text()
+
+    expect(text).toContain("Thank you for submitting your artwork")
+
+    expect(text).toContain("Submit Another Work")
+
+    expect(text).toContain("View Artwork in My Collections")
```

Comment:
```suggestion
    expect(screen.getByText("Submit Another Work").attributes["href"].value).toBe("/bar")

    expect(screen.getByText("View Artwork in My Collections").attributes["href"].value).toBe("/fooooo")
```

Reply:
I try to use it like this 
```
expect(screen.getByText("Submit Another Work")).toHaveAttribute(
      "href",
      "/sell2/submissions/new"
    )
```
And I get back this
```
    Expected the element to have attribute:
      href="/sell2/submissions/new"
    Received:
      null
```
I don't understand what's wrong here. Can you stop a mistake? 👀

Reply:
When I use the syntaxis that you suggested it returns 
``` TypeError: Cannot read properties of undefined (reading 'value') ```

Reply:
Hmm, in [this](https://github.com/artsy/force/blob/f23c8def6659d9408216b1546833ce2dbc92a305/src/Apps/Sell/Routes/__tests__/IntroRoute.jest.tsx#L55) test file, it works. Maybe worth looking into what's different in the test setup (or how the link is constructed).

Reply:
Daaamn I should have used getByTestId
Thanks!